api/config/load_balancer/config_request.go

```diff
@@ -61,6 +61,8 @@ func DefaultConfigRequest() *ConfigRequest {
 	c.V1.Sys.Ngx.Http.LargeClientHeaderBuffersSize = w.String("8k")
 	c.V1.Sys.Ngx.Http.Sendfile = w.String("on")
 	c.V1.Sys.Ngx.Http.SslCiphers = w.String(config.ExternalCipherSuite)
+	c.V1.Sys.Ngx.Http.SslClientCertificateFilename = w.String("root_ca.crt")
+	c.V1.Sys.Ngx.Http.SslVerifyClient = w.String("optional")
```

Comment:
I wonder, while we are here, do we want to change this default to off?  Turning it off will make us compatible with libressl by default and I doubt many people are using client certs at the lb.

Reply:
Since I changed the default for `ssl_verify_client` to off, I also made the default for `ssl_client_certificate_filename` to `""` for consistency.